recipes/trivy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "trivy" %}
+{% set version = "0.55.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aquasecurity/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: dfe18dbe98ce4edecbe15a830840e914f8234d99ac8074f7992c315a1ce98392
+
+
+build:
+  number: 0
+  script:
+    - go install ./cmd/trivy
+    - go-licenses save ./cmd/trivy --save_path licenses --ignore github.com/csaf-poc/csaf_distribution --ignore modernc.org/mathutil
```

Comment:
You would need to add the two ignored licenses manually to the recipe directory.

Reply:
Thanks @xhochy.  I'm not too familiar with this paradigm, let me know if further changes are needed.